src/main/java/Duke.java

```diff
@@ -1,10 +1,110 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+
+    public static Task[] tasks = new Task[100];
+    public static int tasksLength = 0;
+
+    public static void printNewTask(int taskNumber) {
+        Greeting.printSeperator();
+        System.out.println("\tGot it. I've added this task: \n"+ "\t\t" + tasks[tasksLength].printTask());
+        tasksLength ++;
+        System.out.println("\tNow you have " + tasksLength + " tasks in the list.");
+        Greeting.printSeperator();
+    }
+
+    public static void mark(String userInput) {
+        int taskNumber = Integer.parseInt(userInput);
+        tasks[taskNumber - 1].MarkStatusDone();
+        Greeting.printSeperator();
+        System.out.println("\tNice! I've marked this task as done:\n" +
+                "\t\t[X] " + tasks[taskNumber - 1].getTask());
+        Greeting.printSeperator();
+    }
+
+    public static void unmark(String userInput) {
+        int taskNumber = Integer.parseInt(userInput);
+        tasks[taskNumber - 1].MarkStatusUndone();
+        Greeting.printSeperator();
+        System.out.println("\tOK, I've marked this task as not done yet:\n" +
+                "\t\t[ ] " + tasks[taskNumber - 1].getTask());
+        Greeting.printSeperator();
+    }
+
+    public static void ChatPolling() {
+        String userInput;
+
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            userInput = in.nextLine();
+            String[] inputType = userInput.split(" ", 2);
+            //System.out.println(userInput);
+
+            switch (inputType[0]) {
+                case "bye": 
+                    Greeting.printSeperator();
+                    return;
```

Comment:
Do include explicit //Fallthrough comment whenever there is a case statement without a break statement